composer.json

```diff
@@ -10,6 +10,50 @@
     {
       "type": "composer",
       "url": "https://packagist.org"
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "plugins/wpml",
+        "version": "4.3.12",
+        "dist": {
+          "url": "https://storage.googleapis.com/planet4-3rdparty-plugins/sitepress-multilingual-cms.4.3.12.zip",
+          "type": "zip"
+        }
+      }
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "plugins/wpml-management",
+        "version": "2.9.6",
+        "dist": {
+          "url": "https://storage.googleapis.com/planet4-3rdparty-plugins/wpml-translation-management.2.9.6.zip",
+          "type": "zip"
+        }
+      }
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "plugins/wpml-stringtranslate",
+        "version": "3.0.9",
+        "dist": {
+          "url": "https://storage.googleapis.com/planet4-3rdparty-plugins/wpml-string-translation.3.0.9.zip",
+          "type": "zip"
+        }
+      }
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "plugins/ideapush",
+        "version": "7.19",
+        "dist": {
+          "url": "wp plugin install --activate https://storage.googleapis.com/planet4-3rdparty-plugins/ideapush-v7.19.zip",
```

Comment:
This one still has the command in it :wink: Speaking of that, currently `composer-local.json` of NROs also includes the activation script. Not sure if we need to preserve that, as these are already activated on all dev stage and prod.

Reply:
Oops fixed!

In theory `activate` is not needed for plugins that are already activated.